src/chrome/content/rules/Sfu.ca.xml

```diff
@@ -84,7 +84,6 @@
 	<target host="apps.code.sfu.ca" />
 	<target host="beediecommunity.sfu.ca" />
 	<target host="beedie.sfu.ca" />
-	<target host="blogs.sfu.ca" />
```

Comment:
Okay. I simply deleted it because that subdomain seems to have been "stopped", but it's true that blogs.sfu.ca is still reachable.